### PR TITLE
faster codesigning

### PR DESCRIPTION
**Pull request by jhheider**
Testing a few configurations (mesonbuild is our python venv test subject):

find | codesign (@main): 3m17 / 4m8
codesign --deep: `bundle format unrecognized, invalid, or unsuitable`
find bin sbin lib libexec codesign: 1s / 0s
find | file | codesign: \<aborted due to comments below\>
find !.py | codesign: 11s

So, I think the find -not -name version is the winner. We can expand the blacklist if we find ones that run too long, but this is a start:
- .py
- .pyc
- .h
- .txt

### CODE REVIEW

**Comment by jhheider** [comment]
I think this isn't going to work. Once of the files we needed to sign was this:
```sh
$ file ~/.tea/nodejs.org/v19/include/node/common.gypi    
/Users/jacob/.tea/nodejs.org/v19/include/node/common.gypi: ASCII text
```

File: .github/actions/codesign/action.yml [hunk 1]
```diff
@@ -45,21 +45,50 @@ runs:
...
+        # `tea` won't check due to
+        # https://github.com/denoland/deno/issues/17753
+        echo "check<<EOF" >> $GITHUB_OUTPUT
+        /usr/bin/find $PATHS \
+          -type f \
+          -not -name '*.py' \
+          -not -name '*.pyc' \
+          -not -name '*.txt' \
+          -not -name '*.h' \
+          -not -name tea | \
```
Comment (mxcl) [inline .github/actions/codesign/action.yml:71]: Needs a comment because I sure don’t get why we’re skipping ourselves

Comment (jhheider) [reply .github/actions/codesign/action.yml:71]: Comment is right above. Same reason. Deno compiled binaries won't pass strict evaluation.

Comment (jhheider) [reply .github/actions/codesign/action.yml:71]: I can expand it with more context.

Comment (jhheider) [reply .github/actions/codesign/action.yml:72]: More words added.

**Comment by mxcl** [comment]
I’m going LGTM because it's quite deep and I can't grok easily enough

**Comment by jhheider** [comment]
virtualenv:

new: [10s](https://github.com/teaxyz/pantry.core/actions/runs/4369498849/jobs/7643360030)
prev: [164s](https://github.com/teaxyz/pantry.core/actions/runs/4296409005/jobs/7488099836)

@mxcl

**Pull request merged**